CODEOWNERS

```diff
@@ -505,6 +505,7 @@
 /samples/dect/dect_phy/dect_shell/        @nrfconnect/ncs-modem-tre
 /samples/dect/dect_phy/hello_dect/        @nrfconnect/ncs-modem
 /samples/dfu/dfu_target/                  @nrfconnect/ncs-charon
+/samples/dfu/dfu_multi_image/             @nrfconnect/ncs-charon
```

Comment:
Need another entry for /samples/dfu/dfu_multi_image/ *.rst  @nrfconnect/ncs-charon-doc

Reply:
Added

Reply:
Ok done